app/views/admin/songs/_form.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= f.input :title %>
   <%= f.input :author %>
   <%= f.input :melody %>
-  <%= f.input :category %>
+  <%= f.association :song_category, include_blank: false %>
```

Comment:
You can probably use `f.input` here as well. And we might want to consider setting `include_blank: true` and then check if category is blank during save to force the user to select the correct category. Otherwise the user might just not even look at the field and many songs will end up in a random category.